### ISSUE TITLE
Pandas error from getReport2 (for a request with no dimensions and no STATIC_ROW_COMPONENT.

### ISSUE THREAD

**Issue by cyaris** [pitchmuc/adobe-analytics-api-2.0#94]
```
lib/python3.11/site-packages/pandas-1.5.3-py3.11-macosx-12-x86_64.egg/pandas/core/internals/construction.py", line 666, in _extract_index
    raise ValueError("All arrays must be of the same length")
ValueError: All arrays must be of the same length
```

**Comment by pitchmuc** [comment]
Hello @cyaris,
Sorry for the delay, I was working on another wrapper.  
You should always have a dimension set for your report request. 
What happens in the UI is that a default dimension is set for you when you pull a metric. 

You can retrieve the list of dimension by using the `getDimensions(rsid)` method

**Comment by cyaris** [comment]
Hi @pitchmuc, not sure if I'm following. Why do these types of requests run successfully for `getReport2` but not `getReport`? Thanks!

**Comment by pitchmuc** [comment]
Sorry for getting back this late, the code to decrypt the response from the Analytics server and return a dataframe is not the same for getReport vs getReport2. 
The one from getReport2 is much more flexible and structure in a way that I can adapt the decrypt code fairly easily. 

The one in getReport is much more compact and have many interdependent variables. This change would require to completely refactor the code or migrate to the other code, but if that is the case, I would just recommend you to use the getReport2 method. 

Basically, one is the code I wrote 6 years ago and the other one 2 year ago after 4 year of extended coding activities in python, so it is way better structure and developed. 

I hope it helps.
